### PR TITLE
New exit list logic.

### PR DESCRIPTION
**Pull request by dnesbitt61**
Added GraphId to the GraphTileHeader.
This is a new GetExitSignList method. When it is working the existing exit sign logic (with exits stored on EdgeInfo) will be removed.

### CODE REVIEW

File: src/baldr/directededge.cc [hunk 1]
```diff
@@ -24,9 +24,16 @@ GraphId DirectedEdge::endnode() const {
   return endnode_;
 }
 
+// ------------------  Data offsets and flags for extended data -------------//
+
 // Get the offset to the common edge data.
 uint32_t DirectedEdge::edgedataoffset() const {
```
Comment (kevinkreiser) [inline src/baldr/directededge.cc]: i always wondered why this wasnt named edgeinfooffset

Comment (dnesbitt61) [inline src/baldr/directededge.cc]: Good point. Probably a big change but should do it

Comment (kevinkreiser) [inline src/baldr/directededge.cc]: yes a very annoying change that breaks all the builds but might be for the best.

**Comment by dgearhart** [comment]
:+1:

**Pull request merged**